packages/suite/src/components/suite/DeviceDisplay.tsx

```diff
@@ -0,0 +1,221 @@
+import { Icon, variables } from '@trezor/components';
+import { Translation } from './Translation';
+import { ClipboardEvent, ReactNode } from 'react';
+
+const Display = styled.div`
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    border-radius: 16px;
+    min-height: 134px;
+    width: 100%;
+    background: #000000;
+    padding: 25px 12px;
+`;
+
+const Text = styled.div<{ isPixelType: boolean; isWithIndentation?: boolean }>`
+    font-family: ${({ isPixelType }) => (isPixelType ? 'PixelOperatorMono8' : 'RobotoMono')};
+    font-size: ${({ isPixelType }) => (isPixelType ? '12' : '18')}px;
+    color: white;
+    word-break: break-word;
+    max-width: 216px;
+    display: inline;
+    text-indent: ${({ isWithIndentation }) => (isWithIndentation ? '28px' : '0')};
+`;
+
+const Row = styled.div<{ isAlignedRight?: boolean }>`
+    display: flex;
+    align-items: center;
+    gap: 8px;
+    justify-content: ${({ isAlignedRight }) => (isAlignedRight ? 'flex-end' : 'flex-start')};
+`;
+
+const Chunks = styled.div`
+    display: flex;
+    flex-direction: column;
+    gap: 5px;
+`;
+
+const Wrapper = styled.div`
+    position: relative;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+`;
+
+const Divider = styled.div<{ areChunksUsed: boolean }>`
+    width: 100%;
+    height: 1px;
+    background: #2b2b2b;
+    margin: ${({ areChunksUsed }) => (areChunksUsed ? '20' : '0')}px 0;
+`;
+
+const AddressLabel = styled.span<{ areChunksUsed: boolean }>`
+    font-weight: 600;
+    color: #808080;
+    font-size: ${variables.FONT_SIZE.TINY};
+    text-transform: uppercase;
+    position: absolute;
+    background: #000000;
+    padding: 0 10px;
+    text-align: center;
+    left: 50%;
+    bottom: ${({ areChunksUsed }) => (areChunksUsed ? '12' : '-7')}px;
+    transform: translate(-50%, 0);
+`;
+
+const StyledNextIcon = styled(Icon)<{ isPixelType: boolean }>`
+    position: relative;
+    bottom: ${({ isPixelType }) => (isPixelType ? '13' : '20')}px;
+    right: ${({ isPixelType }) => (isPixelType ? '-95' : '35')}px;
+`;
+
+const StyledContinuesIcon = styled(Icon)<{ isPixelType: boolean }>`
+    position: relative;
+    top: ${({ isPixelType }) => (isPixelType ? '10' : '25')}px;
+    right: ${({ isPixelType }) => (isPixelType ? '82' : '97')}px;
+`;
+
+export interface DeviceDisplayProps {
+    address: string;
+    network?: string;
+}
+
+export const DeviceDisplay = ({ address, network }: DeviceDisplayProps) => {
+    const selectedDeviceInternalModel = useSelector(selectDeviceInternalModel);
+    const unavailableCapabilities = useSelector(selectDeviceUnavailableCapabilities);
+    const addressDisplayType = useSelector(selectAddressDisplayType);
+
+    const isCardano = network === 'cardano';
+    const areChunksUsed =
+        addressDisplayType === AddressDisplayOptions.CHUNKED &&
+        !unavailableCapabilities?.chunkify &&
+        !isCardano;
```

Comment:
So it should not be used for cardano or is it a leftover from some experiment?

Reply:
Not used for Cardano now, FW fixed some bugs that they had, waiting for their release but for now we will push it without Cardano

Reply:
Leave a comment maybe? Because in the current version there is some unreachable code.